functions/influxdb+grafana.bats

```diff
@@ -3,16 +3,20 @@
 load influxdb+grafana
 load helpers
 
-@test "destructive-install_influxDB" {
+@test "destructive-influxDB_install" {
   run influxdb_install "Password1234"
   [ "$status" -eq 0 ]
-  run systemctl is-active --quiet influxdb.service
+  echo -e "# \e[32mInfluxDB installation successful." >&3
```

Comment:
I would say this is implicit understood if the test passes. I do not want to flood the CI log, shellcheck does it well already 😆 
What do you have in mind?

Reply:
Actually when working on tests inside docker it's useful to have intermediate output.
This is just 2 lines. Shellcheck is yet another story of course